File: frontend/types/InventoryItem.ts
export type InventoryItem = {
  id: string;
  name: string;
  type: string;
  available_quantity: number;
  unit: string;
  warehouseNamesCritical: string[];
  warehouseNamesWarning: string[];
  warehouseNamesGood: string[];
};

File: backend/src/main/java/com/bamx/backend/dtos/InventoryItem.java
package com.bamx.backend.dtos;

import java.util.List;
import lombok.*;

@Data
@Builder
public class InventoryItem {
  private String id;
  private String name;
  private String type;
  private Double available_quantity;
  private String unit;
  private List<String> warehouseNamesCritical;
  private List<String> warehouseNamesWarning;
  private List<String> warehouseNamesGood;
}
